rules/xcodeproj.bzl

```diff
@@ -323,6 +326,19 @@ def _xcodeproj_impl(ctx):
     xcodeproj_schemes_by_name = {}
 
     for target_info in targets:
+        target_name = target_info.name
+
+        if "xcodeproj-ignore-as-target" in target_info.tags:
```

Comment:
What if we just skipped all targets that had the `manual` tag? If we don't want a target to show up from doing a `bazel query`, do we really want it to show up their xcodeproject? I would guess not.